apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -1404,7 +1416,6 @@ fn request_ftv1(mut req: SubgraphRequest) -> SubgraphRequest {
         .private_entries
         .lock()
         .contains_key::<EnableSubgraphFtv1>()
-        && Span::current().context().span().span_context().is_sampled()
```

Comment:
Should this PR still remove this line if it no longer modifies `Conf::calculate_field_level_instrumentation_ratio` on the telemetry plugin?

Reply:
I think this line should be replaced to check if the span is enabled instead of the opentelemetry specific `.span_context().is_sampled()`

Reply:
I can do the same test as https://github.com/apollographql/router/pull/2894/files#diff-9b15f0e8dbcda44df53d299481383c7b01c32e4db4bb61594092f9389c4748cfR177-R183